src/vector/platform/ElectronPlatform.tsx

```diff
@@ -269,19 +268,19 @@ export default class ElectronPlatform extends VectorBasePlatform {
 
     public startUpdateCheck() {
         super.startUpdateCheck();
-        electron.send('check_updates');
+        window.electron.send('check_updates');
     }
 
     public installUpdate() {
         // IPC to the main process to install the update, since quitAndInstall
         // doesn't fire the before-quit event so the main process needs to know
         // it should exit.
-        electron.send('install_update');
+        window.electron.send('install_update');
     }
 
     public getDefaultDeviceDisplayName(): string {
         const brand = SdkConfig.get().brand;
-        return _t('%(brand)s Desktop (%(platformName)s)', {
+        return _t('%(brand)s Desktop: %(platformName)s', {
```

Comment:
Seems inconsistent to use `on X` on Web, but `: X` on Desktop

Reply:
That was the product decision